bgpd/bgp_attr.h

```diff
@@ -53,6 +53,7 @@
 /* Valid lengths for mp_nexthop_len */
 #define BGP_ATTR_NHLEN_IPV4               IPV4_MAX_BYTELEN
 #define BGP_ATTR_NHLEN_VPNV4              8+IPV4_MAX_BYTELEN
+#define BGP_ATTR_NHLEN_IPV6_LL IPV6_MAX_BYTELEN
```

Comment:
Nit.  tab instead of single space to match surrounding?

Reply:
@donaldsharp  The format of line no 56 is as per git clang-formatting. I tried changing the format just like no 55, however  clang-formatting is still throwing error.